### DISCUSSION TITLE
GitHub Education licence not given despite sending student ID card image

### DISCUSSION THREAD

**Discussion by anja-sehovac** [community/community#112441] [GitHub Education]
### Hi everyone,

I have applied for the academic licence by sending an image of my student ID. I need the licence for my college and the rest of my colleagues who submitted the same got it, but I have not

**Comment by AidenJamesYt** [comment]
how long ago did you submit this

**Comment by samus-aran** [comment]
Hey @anja-sehovac  👋

When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

We can see that your academic benefits have been granted, congratulations! Enjoy! 🎉
